feat: UwUśny feature

Sanielia opened the pull request:
- Added h3gg profile description to README
- Added polish and english version of XXV Anniversary Championships - Warmup tournament regulations and privacy policy

Sanielia committed:
feat: UwUśny feature
- Added h3gg profile description to README
- Added polish and english version of XXV Anniversary Championships - Warmup tournament regulations and privacy policy

Norbiros commented on README.md:
```diff
@@ -1,3 +1,38 @@
+- Keszu - H3gg leader,
+- Berkas, CzileraInżyniera, Vuko & Zuuzel - our socialmedia admins,
+- Kicferk - H3gg website admin,
+- Kubaryt - H3gg Github profile admin,
```
Who is "kubaryt"?

Sanielia replied:
Idk

Norbiros commented on README.md:
```diff
@@ -1,3 +1,38 @@
+Our team consist of many expierenced Heroes III tournament organizers, players and template makers, ATM there are 20 of us:
+
+- Keszu - H3gg leader,
+- Berkas, CzileraInżyniera, Vuko & Zuuzel - our socialmedia admins,
```
I don't know them

Sanielia replied:
Idc

goteusz-maszyk commented on README.md:
```diff
@@ -1,3 +1,38 @@
+> UwU UwU UwU UwU
+> ![](https://media.discordapp.net/attachments/1022538414328913930/1216040283406467162/image.png?ex=66082b44&is=65f5b644&hm=247cc5c49e003dc93d5f4ae2c556061ed35c96e7fecff1051641c66ceefa3451&=&format=webp&quality=lossless&width=550&height=310)
+
+# We are H3gg 👋
```
We're not.

Sanielia replied:
Okok

Norbiros commented on README.md:
```diff
@@ -1,3 +1,38 @@
+## 🐎 About us:
+
+We are group of people, who love Heroes of Might and Magic III, and want to see it flourish.
+We have organized many Heroes III tournaments i.e. XX Anniversary Tournament, World Championships, Polish Cup and XXV Anniversary Championships.
```
I never played heroses 3

Sanielia replied:
Idc

goteusz-maszyk commented:
jednak muszę dać żebyś nie mógł zamykać konwersacji cwaniaczku

tapczanek1321 commented:
co czemu ja to mam reviewowac

tapczanek1321 commented on README.md:
```diff
@@ -1,3 +1,38 @@
-jedzcie dzieci!
\ No newline at end of file
+> [!IMPORTANT]
+> UwU UwU UwU UwU
```
UwU

Sanielia replied:
UwU

Sanielia committed:
Resolving request changes
- changed H3gg to Gwardia Czapli
- added disclaimers about members and Heroes III

Norbiros reviewed (approved):
lgtm!

goteusz-maszyk commented on README.md:
```diff
@@ -1,3 +1,38 @@
+- Vanitasss - H3gg server and bot owner on Discord,
+- Bejker, Fafhrd, KacperOK, Makiki, mvrcxlinv, Niechcic, Ortant, Ponczek, Ravciozo, Regrof, Sokol_53 and Valerin Bizonov, who also do other various things.
+
+## 🌐 Our website
+
+We have created biggest polish platform for organizing Heroes III tournaments.
+Including ongoing tournaments, on our website there are 84 tournaments, where most of them are organized by people outside our team.
+You can check it out https://www.h3.gg/.
+
+If you are interested in organizing Heroes III tournament, you can contact us and we will give you organizer permissions on our website.
+
```
that's not our website

Sanielia replied:
It is

goteusz-maszyk commented on README.md:
```diff
@@ -1,3 +1,38 @@
+
+## ✉️ How to reach us:
+
+- You can contact us on our [Discord server](https://discord.com/invite/WPXRs5aX89),
+- You can mail us: `support@h3.gg`
+- You also can write to us on our [Facebook](https://www.facebook.com/h3ggg/) or on our streams on [Twitch](https://twitch.tv/h3gg) and [Youtube](https://www.youtube.com/@H3ggtournaments)
```
that's not our contact

Sanielia replied:
It is
